src/tokens2.jl

```diff
@@ -66,7 +66,11 @@ end
          ii[2].address == 1?                       2 :
          ii[2].address == 2?                       3 : 1
 
+"""
+    compare(m::SAContainer, s::IntSemiToken, t::IntSemiToken)
 
+Here, `s` and `t` are semitokens for the same container `m`; this function determines the relative positions of the data items indexed by `(m,s)` and `(m,t)` in the sorted order. The return value is `-1` if `(m,s)` precedes `(m,t)`, `0` if they are equal, and `1` if `(m,s)` succeeds `(m,t)`.
```

Comment:
I think you can drop the first part:

~~Here, `s` and `t` are semitokens for the same container `m`; this function~~